var/spack/repos/builtin/packages/py-chalice/package.py

```diff
@@ -27,7 +27,7 @@ class PyChalice(PythonPackage):
     depends_on('py-setuptools', type='build')
     depends_on('py-click@7.0:8.0',           type=('build', 'run'))
     depends_on('py-botocore@1.12.86:2.0.0',  type=('build', 'run'))
-    depends_on('py-typing@3.6.4',            type=('build', 'run'))
+    depends_on('py-typing@3.6.4',            type=('build', 'run'), when='^python@:3.4.999')
```

Comment:
According to the `setup.py`, it should be:
```suggestion
    depends_on('py-typing@3.6.4',            type=('build', 'run'), when='^python@:3.6.999')
```
Does it cause problems for Python 3.5 or 3.6?

Reply:
@aweits thoughts?

Reply:
would have to test to be 100% sure. https://github.com/python/typing/issues/573 We should probably also have a conflicts() in py-typing?